packages/verification-vue-component/src/components/DigitalTwin/DigitalTwin.vue

```diff
@@ -0,0 +1,184 @@
+<template>
```

Comment:
I think this should be more integrated into the existing structure. Probably just a special type of a VerificationItem.